### DISCUSSION TITLE
[ week10 ] R3 : 0 異世界挑戰 - LV10 API 問題

### DISCUSSION THREAD

**Discussion by benben6515** [Lidemy/forum#144] [程式導師實驗計畫]
## 如題
在 [R3 : 0 異世界挑戰](https://r30challenge.herokuapp.com/) 的 [第 10 關](https://r30challenge.herokuapp.com/lv10.php?token=00002211) 中
我使用 node 的 [request 套件](https://www.npmjs.com/package/request) 串了 `https://glacial-everglades-11859.herokuapp.com/api.php`

附上 code : 
```javascript
let request = require('request')

const options = {
  url : 'https://glacial-everglades-11859.herokuapp.com/api.php'
}

request(options, (err, res, body) => {
  // console.log(res)
  console.log(body)
})
```

![Screenshot 2021-06-15 181238](https://user-images.githubusercontent.com/61361198/122037394-1d80b800-ce07-11eb-97aa-a85665a533d2.png)

但是發現得到的 response 是 `no-such-app` ，如下圖 : 

![Screenshot 2021-06-15 181322](https://user-images.githubusercontent.com/61361198/122037294-080b8e00-ce07-11eb-97e8-f6ca984fad35.png)

No such app : https://www.herokucdn.com/error-pages/no-such-app.html)

## 疑問
想請問是哪裡出錯嗎@@
還是真的是 heroku 掛掉XD

---
### BTW
我真的用心在玩 (？)
顏文字我也有去找翻譯，很好玩
![Screenshot 2021-06-15 134416](https://user-images.githubusercontent.com/61361198/122038322-1017fd80-ce08-11eb-935e-6848f0739855.png)

第 5 關，轉址的那關，我也沒有用 `Esc` 偷吃步
![Screenshot 2021-06-15 183602](https://user-images.githubusercontent.com/61361198/122039413-3ee2a380-ce09-11eb-9632-51077867bee7.png)

當然花了很多時間，但很值得，你說是吧

**Comment by aszx87410** [comment]
好像是真的掛了，幫你 tag 這遊戲的管理員 @ishin4554

**Reply by benben6515** [reply]
原來是真的掛了
今天都在懷疑人生 XDD
感謝老師

**Reply by pcchen95** [reply]
遇到同樣問題+1，一直以為是自己哪裡寫錯結果卡了好幾個小時QQ 感謝提問

**Reply by torai55** [reply]
我也卡在這試很久
真的以為自己哪裡寫錯，還用 vs code 的 thunder client 去試XD
